### ISSUE TITLE
[Android] Possible to use env vars in strings.xml for style/item ?

### ISSUE THREAD

**Issue by atinder** [react-native-config/react-native-config#37]
App works fine when using env vars in AndroidManifest.xml but crashes when using env vars inside strings.xml

Edit: Basically my question is related to react-native-facebook-account-kit
I want to fetch following info from .env

```
<style name="LoginThemeYellow" parent="Theme.AccountKit">
    <item name="com_accountkit_primary_color">#f4bf56</item>
    <item name="com_accountkit_primary_text_color">@android:color/white</item>
    <item name="com_accountkit_secondary_text_color">#44566b</item>
    <item name="com_accountkit_status_bar_color">#ed9d00</item>

    <item name="com_accountkit_input_accent_color">?attr/com_accountkit_primary_color</item>
    <item name="com_accountkit_input_border_color">?attr/com_accountkit_primary_color</item>
</style>
```

**Comment by superquest** [comment]
This would be great. Any hints on how to implement this? I could give it a try.

**Comment by prtkkmrsngh** [comment]
Its working. Tested for react-native-facebook-login!

**Comment by joncursi** [comment]
@prtkkmrsngh how'd you get it to work? I'm trying to do the same for `react-native-facebook-login`

**Comment by joncursi** [comment]
I'm trying to do this in a dynamic way using this package:

```xml
  <string name="facebook_app_id">appIDhere</string>
```

**Comment by prtkkmrsngh** [comment]
@joncursi edit strings.xml like this

```xml
<string name="facebook_app_id">@string/FACEBOOK_APP_ID</string>
```

Note: FACEBOOK_APP_ID should be present in .env and if you are using different files for different environment (eg. staging, production) then in them too.

**Comment by alfonsodev** [comment]
This is working on my computer but is not working on Bitrise build system
String.xml
```
<resources>
    <string name="facebook_app_id">@string/FACEBOOK_APP_ID</string>
    <string name="fb_login_protocol_scheme">@string/FACEBOOK_LOGIN_PROTOCOL_SCHEME</string>
</resources>
```
```
:app:processReleaseResources/Users/vagrant/git/android/app/build/intermediates/res/merged/release/values/values.xml:923:36-59: AAPT: No resource found that matches the given name (at 'facebook_app_id' with value '@string/FACEBOOK_APP_ID').
```
Does anybody here use a build system and had similar problems ?

**Comment by fforres** [comment]
Same problem here @alfonsodev 
Locally it's no issue, but on a CI/CD environment (budydbuild in my case) it does not work. 

Did you find any solution?

**Comment by alfonsodev** [comment]
@fforres Yes, in my particular case I'm moving the folder after cloning and `mv` command is not copying hidden file by default, so `.env` file doesn't get copied and therefore the issue. 
The solution was to run `shopt -s dotglob` before `mv` command so that `.env` gets copied over with the src files. 
I hope it helps!

**Comment by sundariragavan** [comment]
Same problem here.Any update?

**Comment by lucianomlima** [comment]
Any news on that?

**Comment by mchjohn** [comment]
Try it
> <resources>
>     <string name="facebook_app_id">{$FACEBOOK_APP_ID}</string>
>     <string name="fb_login_protocol_scheme">{$FACEBOOK_LOGIN_PROTOCOL_SCHEME}</string>
> </resources>

**Comment by Makhdoom-Sharif** [comment]
I am trying to access the string from .env in strings.xml but it shows the error: `resource string/APP_NAME (aka com.APP:string/APP_NAME) not found`
. Any update?

**Comment by LuizCristino** [comment]
Looks like any access to string xml results on not found only on CI. Also looking for a solution

**Comment by tkyr-hh** [comment]
this issue happened to me as well but only on CircleCI pipeline, not happening in local,  I don't know why

**Comment by KFrysztak** [comment]
any update on it? I am also facing the same issue. Locally it works in a perfect way but not on CI.

**Comment by tkyr-hh** [comment]
@KFrysztak for me the issue is that I'm decrypting the env file in the first job of the NPM packages installation, not in the APK build job, and it turns out that the ENV file does not exist because I need it to persist in the workspace, but this is not wise, I've put the env decryption step in the APK build job, before the step of the assemble release step, and it worked as expected.

**Comment by MhtChawlaRutrust** [comment]
> I am trying to access the string from .env in strings.xml but it shows the error: `resource string/APP_NAME (aka com.APP:string/APP_NAME) not found` . Any update?

any workaround?
